install.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+
+GREEN=$'\e[0;32m'
+RED=$'\e[0;31m'
+PURPLE=$'\e[0;35m'
+CYAN=$'\e[0;36m'
+NC=$'\e[0m'
+
+echo "Welcome to ${PURPLE}mentorship-backend${NC} installer!"
+read -p "Press Enter to begin installation"
+
+if [ -d "mentorship-backend" ]; then
+  echo "${RED}! ! ! WARNING ! ! !${NC}"
+  echo "mentorship-backend directory already exists. It ${RED}will be deleted${NC} and replaced with the one pulled from GitHub."
+  read -p "Do you want to proceed? [y/n]: " agree
+  
+  echo $agree
+  
+  if [ "$agree" == "y" ] || [ "$agree" == "Y" ]; then
+    rm -rf mentorship-backend
+    echo "mentorship-backend directory was removed!"
+  else
+    echo "${RED} Setup cancelled by user${NC}"
+    exit 2
+  fi
+fi
+
+echo "Cloning project from GitHub..."
+git clone https://github.com/systers/mentorship-backend.git
+
+cd mentorship-backend
+
+echo "Project cloned successfully!"
+
+read -p "Enter admin account email address: " email
+
+if [ -z ${email} ]; then
+  echo "${RED}Entered email is empty. Please try again.${NC}"
+  exit 1
+fi
+
+if ! [[ "$email" =~ ^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,4}$ ]]
+then
+  echo "${RED}Entered string isn't a correct email address. Please try again.${NC}"
+  exit 1
+else
+  echo "Admin email:${CYAN} $email ${NC}"
+fi
+
+username=$(echo $email | cut -f1 -d"@")
+echo "Username inferred from email is:${CYAN} $username ${NC}"
+
+read -p "Enter password for $email: " password
+
+if [ -z ${password} ]; then
+  echo "${RED}Password is empty. Try again.${NC}"
+  exit 1
+else
+  echo "Password:${CYAN} $password ${NC}"
+fi
+
+read -p "Enter mail server address: " mailserver
+if [ -z ${password} ]; then
+  echo "${RED}Mail server is empty. Try again.${NC}"
+  exit 1
+else
+  echo "Mailserver:${CYAN} $mailserver ${NC}"
+fi
+
+# Install the rest of stuff
+
+echo "Detected python3 version: $(python3 --version)"
+
+echo "Installing virtualenv..."
+pip3 install virtualenv
+
+echo "Creating virtual environment named \"venv\" "
+virtualenv venv --python=python3
+
+source ./venv/bin/activate
+
+echo "Installing dependencies from requirements.txt"
+pip3 install -r requirements.txt
+
+rm .env
+
+echo "export FLASK_ENVIRONMENT_CONFIG=dev" >> .env
+echo "export SECRET_KEY=random_secret_key" >> .env
+echo "export SECURITY_PASSWORD_SALT=random_password_salt" >> .env
+echo "export MAIL_DEFAULT_SENDER=$email" >> .env
+echo "export MAIL_SERVER=$mailserver" >> .env
+echo "export APP_MAIL_USERNAME=$username" >> .env
+echo "export APP_MAIL_PASSWORD=$password" >> .env
+
+echo "${GREEN}Installation has been completed successfully!${NC}"
+echo "${GREEN}You're all set!${NC}"
+echo "${GREEN}> ${NC}Below are some links you might want to check out${GREEN} <${NC}"
+echo "Project's GitHub: https://github.com/systers/mentorship-backend"
+echo "Wiki page: https://github.com/systers/mentorship-backend/wiki"
+echo "Our Zulip chat: https://systers-opensource.zulipchat.com/"
```

Comment:
Zulip is a temporary solution more specifically around GCI. If we are adding this info, better link to slack.

Reply:
thanks, I'll change it